### PR TITLE
Overview page

### PR DESCRIPTION
**Pull request by joemcelroy**
- Wrote up overview page
- added a categories autocomplete suggestions feature
- updated the product listing page to be "dynamic" (based on the url)
- 
image

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _docs-elastic-co_ successfully!

* https://docs-elastic-21c06a30d-elastic-dev.vercel.app

* Built with commit 3aec9b87f25674ed2f8b1597332932e893f4632f

> Issues? Visit #next-docs in Slack

File: docs/solutions-ecommerce.mdx [hunk 1]
```diff
@@ -7,7 +7,26 @@ date: 2022-04-13
 tags: ["ecommerce solution", "ecommerce search ui", "overview"]
 ---
 
-- talk about how to build great ecommerce experiences relies on having:
-  - great autocomplete
-  - Product Listing Pages (e.g. /category/clothing)
-  - Recommending products within product detail pages (e.g. /product/clothing/polo-shirt)
+In ecommerce, to build great search experiences for your customers, you need to focus on three major experiences:
```
Comment (yakhinvadim) [inline docs/solutions-ecommerce.mdx:10]: This sentence mentions three parts, but below we describe four, and we have five pages in the navigation. Do you think we should have five items everywhere?

I also think the image is a bit out of place here — it interrupts the text in the middle and brings unrelated concepts, like highlighting. Maybe we could put it in a separate section?

File: examples/sandbox/src/pages/ecommerce/components/Navigation/index.js [hunk 2]
```diff
@@ -214,7 +217,16 @@ function Navigation(props) {
...
+                // eslint-disable-next-line
+                if (!suggestion.name && suggestion.category) {
+                  window.location.href =
+                    "/ecommerce/category/" + suggestion.suggestion;
+                }
+                defaultHandler(suggestion);
+              }}
               onSubmit={(searchTerm) => {
+                // eslint-disable-next-line no-debugger
                 window.location.href = "/ecommerce/search?q=" + searchTerm;
```
Comment (yakhinvadim) [inline examples/sandbox/src/pages/ecommerce/components/Navigation/index.js:230]: Do we need eslint-ignores here? I don't get errors when I remove them.

File: examples/sandbox/src/pages/ecommerce/Category.js [hunk 3]
```diff
@@ -41,13 +41,14 @@ const categoryPageconfig = {
...
+});
 
-export default function CategoryPageTv() {
+export default function CategoryPage(props) {
+  const category = props?.match?.params.category;
   return (
     <>
       <Navigation />
-      <SearchProvider config={categoryPageconfig}>
+      <SearchProvider config={categoryPageconfig(category)}>
```
Comment (yakhinvadim) [inline examples/sandbox/src/pages/ecommerce/Category.js:51]: This page has some TV-specific facets (below). In my PR, I also added some TV-specific fields in the config. If we go forward with a general category page, we should probably move those somewhere.

**Pull request merged**